recipes/filequery/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "filequery" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/filequery-{{ version }}.tar.gz
+  sha256: 3535d1211067853b521cebef5900a0b34cd55adc8ac36a6a8e7af0036948f679
+
+build:
+  entry_points:
+    - filequery = filequery:fq_cli_handler
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8
+    - python-duckdb ==0.9.1
+    - importlib-metadata ==6.8.0
+    - linkify-it-py >=2.0.2
+    - markdown-it-py ==3.0.0
+    - mdit-py-plugins ==0.4.0
+    - mdurl >=0.1.2
+    # numpy 1.26.1 not available on Conda Forge, loosening pin here
```

Comment:
I see upstream pins all dependencies down to the micro version :-(

Reply:
I hate it when they do that, but some folks are very paranoid and maybe they are justified; python packaging being what they are

Reply:
It is not only packaging. The ecosystem is "more accessible" nowadays, one can Google, find "a repo," and report "problems" to anyone. The quotes there is b/c very few actually take the time to: find the right repo, study the problem and make a useful report and, more important than everything else, be polite with the fellow developer.

Sorry for the rant but people to try to defend from this by pinning will ended up falling in the same trap and getting the same bad error reports later on...